crates/sui-core/src/authority.rs

```diff
@@ -218,14 +221,15 @@ pub struct AuthorityState {
     pub secret: StableSyncAuthoritySigner,
 
     /// Committee of this Sui instance.
-    pub committee: Committee,
+    pub committee: ArcSwap<Committee>,
```

Comment:
Are we sure we want to hot swap the committee, instead of creating a new authority object? A lot of places in our code assume this does not change under their feet.